### ISSUE TITLE
"Element not found" when trying to start WSL after latest windows update (build 19042)

### ISSUE THREAD

**Issue by thjwong** [microsoft/WSL#5936]
# Environment
Microsoft Windows 10 Pro
Version	10.0.19042 Build 19042
Debian
WSL 2

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Your Distribution version: [On Debian or Ubuntu run `lsb_release -r` in WSL]
Whether the issue is on WSL 2 and/or WSL 1: [run `cat /proc/version` in WSL]
```

# Steps to reproduce
start wsl via windows terminal
or
simply type "wsl" to start WSL in command prompt

**WSL logs**: 

#  Expected behavior
WSL starts and Debian starts

# Actual behavior
It just displays
"Element not found"

**Comment by Dcros** [comment]
> /dupe #5880

This is not fixed... still hapening
![image](https://user-images.githubusercontent.com/37849856/140487354-125430d2-9a50-4634-81ca-3ebed7e713d3.png)


I tried all the possible solutions up to google 25 ... none worked, I made sure I had all the requirements, I also deleted almost everything from% appdata% and subfolders, uninstalled absolutely everything related or not ... I just have to go back to a version of windows from 2004? that is not solving a problem. Does wsl2 work or not? to find alternatives if it is not functional or is in beta.

TL:DR #5880 soultion its go back windows 10 from 2004, that its not solved.

**Comment by Dcros** [comment]
![image](https://user-images.githubusercontent.com/37849856/140487947-c6363699-1319-4a71-927b-72e15ca7a962.png)
